hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -1212,7 +1212,6 @@ public void testDatabaseLocationWithSlashInWarehouseDir() {
 
   @Test
   @Override
-  @Disabled("Hive currently returns an empty list instead of throwing a NoSuchNamespaceException")
   public void testListNonExistingNamespace() {
```

Comment:
you can remove the entire method here, since it only existed in this class because of the required override in order to disable it